stream-compositions/api/service-api/product-composition-api/src/main/resources/service-api-v2.yaml

```diff
@@ -114,14 +114,16 @@ components:
     ProductPushIngestionRequest:
       type: object
       properties:
-        productGgroup:
+        productGroup:
```

Comment:
The input for a push style ingestion should mimic the response from the pull integration service. I.e. if multiple product groups are being supported in the Integration for PULL, they should also be supported for the input object of PUSH. So ideally this should also be an array of productGroup.

Reply:
Nevermind. Just saw that we don't support push for legalEntity yet
`public Mono<ProductIngestResponse> ingestPush(ProductIngestPushRequest ingestPushRequest) {
        throw new UnsupportedOperationException();
    }`

Reply:
Yeah. Next time :)